latex/Specific Aims.tex

```diff
@@ -104,14 +104,14 @@
 A new generation of physical techniques, alchemical free energy calculations, are poised to fill this void by providing a quantitative, predictive tool that can be used in multiple stages of the drug discovery pipeline, including lead optimization to improve affinity and selectivity or the retention of potency as other physical properties are optimized. 
 
-Recent success of alchemical methods sparked considerable enthusiasm, but the domain of applicability of these techniques is currently highly limited; for broad application and routine use, they need further evaluation, refinement, and development. 
+Recent success of alchemical methods in predicting accurate affinities has sparked considerable enthusiasm, but the domain of applicability of these techniques is currently highly limited; broad application and routine use will require further evaluation, refinement, and development. 
 There is a vast gulf between targets within the domain of applicability and those which are outside it. 
-To rapidly advance, we need carefully selected systems of intermediate complexity to bridge this gulf. 
-Without such a bridge, these techniques may encounter the same problems faced by docking and related techniques: routine failure without clear insights into why, and years to decades spent in making small methodological modifications without dramatic improvements in predictive power.
+Expanding this domain will require focused work on carefully selected systems of intermediate complexity to bridge this gulf. 
+Without such a bridge, these techniques may encounter the same problems faced by docking and related techniques: routine failure without clear insights into why, and years to decades spent making small methodological modifications without dramatic improvements in predictive power.
 
-Here, we will generate data and conduct blind prediction challenges to expand the domain of applicability and robustness of these techniques. 
-The data we generate will span the range between systems which are tractable with current methods to those only slightly less complex than pharmaceutical drug targets covered by the NIH-funded D3R effort, which runs blind challenges on protein-ligand datasets obtained from the pharmaceutical industry.
-This systematic set of blind prediction challenges will expand the domain applicability, pushing free energy techniques into standard application in drug design.
+We propose to collected targeted experimental datasets, use them to conduct blind prediction challenges, and release curated benchmark sets in a manner designed to drive expansion of the domain of applicability and robustness of physical modeling techniques. 
+The data we generate will bridge the untamed chasm from systems tractable with current methodologies to the pharmaceutically-relevant drug targets featured in the NIH-funded D3R effort, which fields blind challenges using protein-ligand datasets from pharma.
```

Comment:
I'll merge, though "untamed chasm" is perhaps a bit over the top; I'll come up with an alternative phrasing that backs off from this a bit. :)